ERROR - 2024-06-07 06:08:31 --> mysqli_sql_exception: Unknown column 'ame' in 'field list' in C:\xampp\htdocs\intern\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\intern\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `in...', 0)
#1 C:\xampp\htdocs\intern\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\xampp\htdocs\intern\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\xampp\htdocs\intern\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\xampp\htdocs\intern\app\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\intern\system\CodeIgniter.php(933): App\Controllers\Home->addData()
#6 C:\xampp\htdocs\intern\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\intern\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\intern\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\intern\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\intern\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-06-07 06:08:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ame' in 'field list'
[Method: POST, Route: Home/addData]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (:ame:, :mobile_number:, :email:, :created_on:)', [...], false)
 2 APPPATH\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-07 06:08:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ame' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'5757565\', \'abc@gmail.com\', \'2024-06-07 06:08:31\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'5757565\', \'abc@gmail.com\', \'2024-06-07 06:08:31\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (:ame:, :mobile_number:, :email:, :created_on:)', [...], false)
 4 APPPATH\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-07 06:08:31 --> [Caused by] mysqli_sql_exception: Unknown column 'ame' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'5757565\', \'abc@gmail.com\', \'2024-06-07 06:08:31\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'5757565\', \'abc@gmail.com\', \'2024-06-07 06:08:31\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'5757565\', \'abc@gmail.com\', \'2024-06-07 06:08:31\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (:ame:, :mobile_number:, :email:, :created_on:)', [...], false)
 5 APPPATH\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-07 06:09:10 --> mysqli_sql_exception: Unknown column 'ame' in 'field list' in C:\xampp\htdocs\intern\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\intern\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `in...', 0)
#1 C:\xampp\htdocs\intern\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\xampp\htdocs\intern\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\xampp\htdocs\intern\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\xampp\htdocs\intern\app\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\intern\system\CodeIgniter.php(933): App\Controllers\Home->addData()
#6 C:\xampp\htdocs\intern\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\intern\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\intern\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\intern\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\intern\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-06-07 06:09:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ame' in 'field list'
[Method: POST, Route: Home/addData]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (:ame:, :mobile_number:, :email:, :created_on:)', [...], false)
 2 APPPATH\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-07 06:09:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'ame' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'9868576\', \'abc@gmail.com\', \'2024-06-07 06:09:10\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'9868576\', \'abc@gmail.com\', \'2024-06-07 06:09:10\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (:ame:, :mobile_number:, :email:, :created_on:)', [...], false)
 4 APPPATH\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-07 06:09:10 --> [Caused by] mysqli_sql_exception: Unknown column 'ame' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'9868576\', \'abc@gmail.com\', \'2024-06-07 06:09:10\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'9868576\', \'abc@gmail.com\', \'2024-06-07 06:09:10\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (\'gfg\', \'9868576\', \'abc@gmail.com\', \'2024-06-07 06:09:10\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `interntable` (`ame`, `mobile_number`, `email`, `created_on`) VALUES (:ame:, :mobile_number:, :email:, :created_on:)', [...], false)
 5 APPPATH\Controllers\Home.php(25): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
